template/order/metabox-form.latte

```diff
@@ -1,29 +1,34 @@
 {varType Packetery\Nette\Forms\Form $form}
 {varType Packetery\Core\Entity\Order $order}
+{varType bool $showHdWidget}
 {varType bool $isCodPayment}
+{varType bool $allowsAdultContent}
+{varType bool $requiresSizeDimensions}
 
 <div data-packetery-order-metabox>
 {formContext $form}
         <div class="packetery-table">
             <input n:name=packetery_order_metabox_nonce />
             <input n:name=packetery_original_weight />
             {foreach [
-                Packetery\Module\Order\Metabox::FIELD_WEIGHT,
-                Packetery\Module\Order\Metabox::FIELD_LENGTH,
-                Packetery\Module\Order\Metabox::FIELD_WIDTH,
-                Packetery\Module\Order\Metabox::FIELD_HEIGHT,
-                Packetery\Module\Order\Metabox::FIELD_ADULT_CONTENT,
-                Packetery\Module\Order\Metabox::FIELD_COD,
-                Packetery\Module\Order\Metabox::FIELD_VALUE,
+                'packetery_weight',
```

Comment:
Proč jsi zde přestal používat konstanty ?

Reply:
Neměl jsem správně nastavený lint a hlásilo mi to tam jako chybu, i když to fungovalo. Vše vráceno, opraveno a funkční.